### ISSUE TITLE
please keep parameter types and return value in definitions when they are interfaces or enums

### ISSUE THREAD

**Issue by ZeeD** [rokucommunity/brighterscript#756]
When `bsc` emit the definitions it seems that both `enum`s and `interface`s are missing.

starting from this `example.bs`

```brighterscript
interface Foo
    field as string
end interface

enum Bar
    value
end enum

function baz(parameter as Foo) as Bar
    return Bar.value
end function
```

the `example.d.bs` file generated is

```brighterscript
interface Foo
    field as string
end interface

enum Bar
    value
end enum
function baz(parameter as object) as object
end function
```

now `baz` has lost the type definition and return value information. 

I'm using
```bash
$ npx bsc --version
0.60.6
```

**Comment by TwitchBronBron** [comment]
Nice catch! I just put up a PR to address this.

**Comment by TwitchBronBron** [comment]
This has been fixed in v0.61.1
